src/main/java/com/blackducksoftware/ohcount4j/Ohcount.java

```diff
@@ -66,6 +66,8 @@ public static void main(String[] args) {
                 annotate(files, getFilenames(files));
             } else if (opts.detect) {
                 detect(files, getFilenames(files));
+            } else if (opts.interactive) {
```

Comment:
The switch is really the "individual language" switch.  Let's change this from `opts.interactive` to `opts.individual`